.github/workflows/ci.yml

```diff
@@ -6,6 +6,9 @@ on:
       - "**"
     tags:
       - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+    branches:
+      - main
```

Comment:
What does this do, as we already run on every push?

Reply:
It is needed to comment the Coverage report to pull request. Otherwise it will be posted to push commit.